hx_lti_initializer/static/AController.js

```diff
@@ -21,4 +21,5 @@ window.AController = window.AController || function(options) {
 		AController.dashboardObjectController = new AController.DashboardController(options.dashboardControllerOptions, options.commonInfo, AController.dashboardView);
 	}
 	AController.main = new AController.AnnotationMain(options);
+	AController.utils = new AController.Utils('https://edge.edx.org');
```

Comment:
Maybe this could be pulled from the `options`? That would make this a little more configurable if we wanted to implement a logger for canvas or something.